api/models.py

```diff
@@ -14,3 +14,22 @@ def __str__(self):
         """one-line docstring for representing the Profile object."""
         return f'{self.user.first_name}'
 
+
+
+
+
+class add_loan_record(models.Model):
+    loan=models.CharField(max_length=200, null=True)
+    user_id=models.ForeignKey(User,on_delete=models.CASCADE)
+    due_date=models.DateField(auto_now=True)
```

Comment:
Duedate should not be auto_now. It should be specifically set by the user

Reply:
0k